lib/palettes.hpp

```diff
@@ -55,7 +56,11 @@ class PaletteState
    { return palette_tex[curr_palette][use_smooth]; }
    GLuint GetAlphaTexture() const { return alpha_tex; }
    void GenerateAlphaTexture(float matAlpha, float matAlphaCenter);
+
+   int NumPalettes();
```

Comment:
Conversion from `size_t` to `int`. I would choose one consistently.

Reply:
changed to `int` throughout. Some of the `PaletteState` methods like `GetSize` use `-1` as a check.